jib-gradle-plugin/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Invalidates Jib's local application layer cache (`<project root>/build/jib-cache` by default) due to changes to caching mechanism. (In other words, the local cache generated by old Jib versions is incompatible with this version.) ([#2499](https://github.com/GoogleContainerTools/jib/pull/2499))
```

Comment:
Seems like this is a little wordy?

Previously cached local application layers(`<project root>/build/jib-cache`) will be ignored because of changes to the caching selectors. (#2499) ?